modules/mobile/docs/openapi.yaml

```diff
@@ -766,6 +895,67 @@ components:
       description: Authentication is via an access token returned from IAM's SSOe service. Pass the access token in the 'Authorization' Header in the format 'Bearer {accessToken}'. e.g. 'Bearer abcdefg1234567'.
 
   schemas:
+    UpdatePaymentInfoRequest:
+      required:
+        - accountNumber
+        - accountType
+        - financialInstitutionName
+        - financialInstitutionRoutingNumber
+      properties:
+        accountNumber:
+          type: string
+          example: "12345678"
+        accountType:
+          type: string
+          example: "Savings"
+          enum:
+            - Savings
+            - Checking
+        financialInstitutionName:
+          type: string
+          example: "PACIFIC PREMIER BANK"
+        financialInstitutionRoutingNumber:
+          type: string
+          example: "021000021"
+    PaymentInfo:
+      required:
+        - id
+        - type
+        - attibutes
+      properties:
+        type:
+          type: string
+          example: "paymentInformation"
+        id:
+          type: string
+          example: "abe3f152-90b0-45cb-8776-4958bad0e0ef"
+        attributes:
+          required:
+            - paymentAccount
+          properties:
+            paymentAccount:
+              type: object
+              required:
+                - accountNumber
+                - accountType
+                - financialInstitutionName
+                - financialInstitutionRoutingNumber
+              properties:
+                accountNumber:
+                  type: string
+                  example: "************6464"
+                accountType:
+                  type: string
+                  example: "Savings account"
+                  enum:
+                    - Savings account
+                    - Checking account
+                financialInstitutionName:
+                  type: string
+                  example: "PACIFIC PREMIER BANK"
+                financialInstitutionRoutingNumber:
+                  type: string
+                  example: "*****9982"
```

Comment:
I think ABA routing numbers are public, and do not need to be obscured.

Reply:
Fixed.